src/kernel/src/process/session.rs

```diff
@@ -1,18 +1,43 @@
-use std::num::NonZeroI32;
+use crate::{fs::Vnode, tty::Tty};
+use gmtx::*;
+use std::{num::NonZeroI32, sync::Arc};
 
 /// An implementation of `session` structure.
 #[derive(Debug)]
 pub struct VSession {
-    id: NonZeroI32, // s_sid
-    login: String,  // s_login
+    id: NonZeroI32,               // s_sid
+    login: Gutex<String>,         // s_login
+    vnode: Option<Vnode>,         // s_ttyvp
+    tty: Gutex<Option<Arc<Tty>>>, // s_ttyp
+    gg: Arc<GutexGroup>,
```

Comment:
We don't need to keep this if it is not using later on.

Reply:
I didn't realize we don't have to hold on to it, because it's an Arc :)

Reply:
Fixed